### ISSUE TITLE
Does not work with Electron Vue

### ISSUE THREAD

**Issue by rnleal** [Hubertformin/electron-pos-printer#10]
I get TimedOut even if applied the suggestion here
https://github.com/Hubertformin/electron-pos-printer/issues/2

Im using Electron Vue with https://nklayman.github.io/vue-cli-plugin-electron-builder/

See full gist here
https://gist.github.com/rnleal/3abce4ef3b762243b8097d2893970685

**Comment by amaury-tobias** [comment]
I use with `vue-cli-plugin-electron-builder` you try adding this on your vue.config.js
```
module.exports = {
  pluginOptions: {
    electronBuilder: {
      externals: ['electron-pos-printer'],
      nodeIntegration: true,
    },
  }
}
```
Works for me.

**Comment by Hubertformin** [comment]
Does the print job show up in the printing queue?
If yes, then it's probably a problem with a printer driver or the connection to the printer.

**Comment by rnleal** [comment]
@amaury-tobias I did this but still get the `timedOut` error. Newb question: where to put the vue.config.js? is it on the root? or inside the /src directory?

**Comment by rnleal** [comment]
@Hubertformin nothing shows up in the printing queue

**Comment by amaury-tobias** [comment]
> @amaury-tobias I did this but still get the `timedOut` error. Newb question: where to put the vue.config.js? is it on the root? or inside the /src directory?
Put on root folder

**Comment by rnleal** [comment]
This was resolved by upgrading Electron to ^9.0
